lib/matplotlib/tests/test_figure.py

```diff
@@ -509,7 +509,7 @@ def test_invalid_figure_add_axes():
         fig.add_axes((.1, .1, .5, np.nan))
 
     with pytest.raises(TypeError, match="multiple values for argument 'rect'"):
-        fig.add_axes([0, 0, 1, 1], rect=[0, 0, 1, 1])
+        fig.add_axes((0, 0, 1, 1), rect=[0, 0, 1, 1])
```

Comment:
`rect` should also be a tuple, though I'm surprised you don't get a type error for invalid parameters here (though that's on purpose and should be ignored.)

Reply:
This is because I identify one type of error and then do regex-based search and replace for that on all files.